R/database.R

```diff
@@ -1225,14 +1223,24 @@ get_manta_sv = function(min_vaf = 0.1,
   }
 
    if(from_flatfile){
-    all_sv = get_combined_sv(projection = projection)
+    all_sv = get_combined_sv(projection = projection) %>%
+      mutate(FILTER = "PASS") #i.e all variants returned with get_combined_sv() all have PASS in the FILTER column.
```

Comment:
I suggest this mutate step be added into get_combined_sv to make it more self-documenting to users that everything coming from that function already has been selected to meet this criteria. One slightly misleading thing (either way) is that the non-Manta calls coming from get_combined_sv didn't actually "PASS" the same filter, right?

Reply:
Yes, I can move this. Sounds reasonable to me. Without knowing the exact filters for each variant caller, I would not assume so. But I can look into it further if we need to know this.

Reply:
I don't think that column even exists for non-Manta outputs (i.e. GRIDSS)